google_metadata_script_runner/main.go

```diff
@@ -162,9 +162,7 @@ func downloadScript(ctx context.Context, path string, file *os.File) error {
 			return nil
 		}
 
-		if err != nil {
-			logger.Infof("Failed to download object [%s] from GCS bucket [%s], err: %+v", object, bucket, err)
-		}
+		logger.Infof("Failed to download object [%s] from GCS bucket [%s], err: %+v", object, bucket, err)
```

Comment:
Can you add the context that we'll try unauth download to the log msg?

Reply:
There's already a log below this which state that -  `Trying unauthenticated download` (line 167)